src/database/index.tsx

```diff
@@ -174,6 +174,49 @@ export const getBaselineScenarioRef = async () => {
   return results.docs[0].ref;
 };
 
+export const saveScenario = async (scenario: {}): Promise<void> => {
+  try {
+    // We're cheating here because for the launch we know there is only a
+    // baseline scenario. In subsequent launches, we'll need to pass in
+    // the ID of the specific scenario that we want to save. See:
+    // https://github.com/Recidiviz/covid19-dashboard/issues/186
```

Comment:
https://github.com/Recidiviz/covid19-dashboard/issues/129 is still the one that's applicable for this particular comment

Reply:
oops! will update